executor/src/witgen/processor.rs

```diff
@@ -662,15 +662,15 @@ Known values in current row (local: {row_index}, global {global_row_index}):
             .process_identity(identity, &row_pair)
             .is_err()
         {
-            log::debug!(
+            log::trace!(
```

Comment:
This only means that we stop proposing rows and resort to the normal solving, which is a *very* soft failure and is normal to happen in the last row of a normal VM.